bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx60.html

```diff
@@ -0,0 +1,124 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% from "macros.html" import google_play_button, send_to_device with context %}
+
+ {% add_lang_files "firefox/whatsnew_59" "firefox/mobile" "firefox/sendto" %}
+
+ {% extends "firefox/whatsnew/base-pebbles.html" %}
+
+ {% block page_title %}{{ _('See what’s new with Firefox') }}{% endblock %}
+
+ {% block body_id %}firefox-whatsnew{% endblock %}
+ {% block body_class %}{% endblock %}
+
+ {% block site_header %}{% endblock %}
+
+ {% block page_css %}
+   {% stylesheet 'firefox_whatsnew_60' %}
+ {% endblock %}
+
+ {% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+ {% block content %}
+ <main>
+   {% with custom_message=_('Congrats! You’re using the latest version of Firefox.') %}
+     {% include 'firefox/whatsnew/includes/header.html' %}
+   {% endwith %}
+   <div class="main-content {% if show_send_to_device %}has-send-to-device{% else %}no-send-to-device{% endif %}">
+     <div class="content-container visible" id="logo-container">
+       <div class="logo-wrapper">
+         {{ high_res_img('logos/firefox/logo-quantum.png', {'alt': 'Firefox', 'width': '100', 'height': '100', 'class': 'logo-fx showing', 'id': 'logo-fx'}) }}
```

Comment:
The Firefox logo here has a class of `showing` by default, which means it's the only thing visible before the JS executes. This making for a very quick but noticable content jump on page load:

https://screenshots.firefox.com/JyJviwyhEwH0s2pp/127.0.0.1

Would hiding it by default along with the copy make sense, or cause more trouble than it's worth?

Reply:
Good suggestions. Will do.